### PR TITLE
Update representatives

### PR DESCRIPTION
**Pull request by awsmit07**
Update PASA representatives to reflect changes from the cubby brawl election:
- Add Khadeeja as first year rep
- Update Emma to second year rep
- Update Emina to Astrophysics representative
- Remove Mckinley

### CODE REVIEW

File: html/about.html [hunk 1]
```diff
@@ -225,13 +227,13 @@ <h3>Executive Team</h3>
                                     <span>VP Events</span>
                                 </div>
                                 <div class="col col-5 col-sm-3 avatar">
-                                    <div id="emma" class="av-img"></div>
-                                    <span><strong>Emma</strong></span>
-                                    <span>First Year Representative</span>
+                                    <div id="khadeeja" class="av-img"></div>
+                                    <span><strong>Khadeeja</strong></span>
+                                    <span>Second Year Representative</span>
```
Comment (marcus-k) [inline html/about.html:232]: Should be first year rep.

**Review by marcus-k — CHANGES_REQUESTED** [review]
After the first year change, looks all good.

**Comment by awsmit07** [comment]
> After the first year change, looks all good.

Should be fixed

**Review by marcus-k — APPROVED** [review]
> > After the first year change, looks all good.

sounds good

**Pull request merged**